src/main/java/com/aws/greengrass/mqtt/bridge/clients/MessageClient.java

```diff
@@ -37,7 +37,7 @@
 
     T convertMessage(Message message);
 
-    void start();
+    void start() throws MessageClientException;
```

Comment:
so if start fails, what are we doing? Retry with backoff?

Reply:
looking more closely MQTTClient won't actually throw on start.....method had throws on it.   going to do some more cleanup

Reply:
what are the reasons why start can fail then?

If bridge can't connect to the local broker we shouldn't be erroring. We relatively commonly see in our UATs that the bridge takes a few attempts to connect.

Reply:
There's a keystoreexception which can't actually be thrown....

Reply:
connect happens async, and if exception happens yeah it retries w/ backoff